packages/cf-component-tabs/src/Tabs.js

```diff
@@ -3,6 +3,17 @@ const { PropTypes } = React;
 const Viewport = require('cf-component-viewport');
 const Select = require('cf-component-select');
 
+const find = (list, condition) => {
+  let foundElement = undefined;
+  list.forEach(element => {
+    if (condition(element)) {
+      foundElement = element;
+      return;
```

Comment:
I don't think you can short-circuit `forEach` like this, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#Description.